clients/mssql/values.go

```diff
@@ -54,9 +53,12 @@ func parseValue(colVal any, colKind columns.Column, additionalDateFmts []string)
 			}
 
 			colValString = string(colValBytes)
-		} else {
-			// Else, make sure we escape the quotes.
-			colValString = stringutil.Wrap(colVal, true)
```

Comment:
This is not needed.

Reply:
Nice!

Reply:
Do we need to insert a JSON exceeded value marker if the colVal is an array or JSON object?

Reply:
I don't think so. This branch of logic already established that this column is string-based (or else it'd be in ARRAY or STRUCT).